woocommerce/wc-admin
# Change from MenuGroup to label

Fixes #3642 

Change from MenuGroup to using checkbox and radio inputs

This PR removes icons and uses the default input styling.  @Aljullu does this style work for you?

### Accessibility
-   [x] I've used Axe extension

### Screenshots
<img width="566" alt="checkbox" src="https://user-images.githubusercontent.com/56378160/104796042-120a6180-5781-11eb-901a-2e7c7f4e2af1.png">
<img width="561" alt="radio" src="https://user-images.githubusercontent.com/56378160/104796043-12a2f800-5781-11eb-90a7-554701975999.png">

### Detailed test instructions:

1. Run storybook.
2. Add Single knob and check if renders as expected
